CHANGELOG.md

```diff
@@ -72,9 +72,13 @@ To release a new version, please update the changelog as followed:
 - Repository:
   - Danger CI has been added to enforce the update of the changelog (by @lgarithm and @DEKHTIARJonathan)
 - Tutorials:
-  - `tutorial_tfslim` added by (@2wins).
+  - `tutorial_tfslim` added (by @2wins).
+- Tests:
+  - `test_utils_predict.py` added to reproduce and fix issue #288 (by @2wins).
 
 ### Changed
+- Util:
+  - `predict` function modified to fix #565 (by @2wins).
```

Comment:
If it is a bug fix => It should go to the section. [Fixed](https://github.com/2wins/tensorlayer/blob/5ccc7a912331cc5061a7722af0f448049d0a941b/CHANGELOG.md#fixed)

I would do something like this:
```
- Issue #288 related to `tl.utils.predict` Fixed - predict np.vstack problem when batch_size if no factor of test dataset size
```